3.workload.md

```diff
@@ -0,0 +1,378 @@
+# 3. Workload
+
+This section defines the workload definition schematics.
+
+Workload definitions are provided by the platform (just as traits are) so that users may inspect the platform and
+ learn what workloads are available for use. Workload definitions are not extensible to platform users (only to platform
+  operators). Thus, platform users MUST NOT be allowed to create new workload kinds.
+
+## Top-Level Attributes
+
+The following attributes are common across all schemata defined in this document. This structure is designed to allow
```

Comment:
I hope we can delete this section from every doc, and link/copy Kubernetes resource model documentation here (https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields).

Reply:
This only happens once.  Also, I thought  that our GKV are "inspired" by the K8s but  not direct dependent on K8s definition.

Reply:
Strongly agree. :) Though we might consider linking instead to the API conventions document; I believe this is the closest thing that exists to a formal specification of the Kubernetes API.

https://github.com/kubernetes/community/blob/1d6d124/contributors/devel/sig-architecture/api-conventions.md

Reply:
Yes, the api-conventions.md doc is the best candidate here. It also reminds me that referring to an epoch design doc of Kube feels much better than linking K8s doc. Good point!

To @ryanzhang-oss , for metadata and resource model, I'd suggest we directly use it instead of "inspired". "Inspire" is not good unless we significantly improved it and decide not contribute back. 

If needed (seems not IMO), we can clearly clarify that we only support a certain subset of the whole k8s resource model convention. But I'd strongly prefer eliminating the feeling of "inspire".